clients/include/testing_matmul.hpp

```diff
@@ -364,6 +364,16 @@ void check(hipStream_t                          stream,
     }
 }
 
+void check_bias_type(const Arguments& arg)
+{
+    if((arg.a_type == HIP_R_8F_E4M3_FNUZ || arg.a_type == HIP_R_8F_E5M2_FNUZ)
+        && (arg.b_type == HIP_R_8F_E4M3_FNUZ || arg.b_type == HIP_R_8F_E5M2_FNUZ))
+    {
+        CHECK_SUCCESS((arg.bias_type == arg.scale_type || arg.bias_type == arg.d_type) && "For hipblastlt-bench/test, "
+                       "bias_type must be explicitly specified as compute_type or d_type when input A/B are 8bits-typed float for now.");
+    }
+}
+
 template <typename TiA, typename TiB, typename To, typename Tc, typename Tci>
```

Comment:
I would like to add a "determine_bias_type" function which determines the default type / checks if type is valid.
For default bias_type (when it is unset), it should be aligned with #598 
But I think that probably needs more template designs. So I'd left it to the other independent PR.
For this PR, I only check "**When running hipblastlt-bench/test**, bias_type must be explicitly specified as compute_type or d_type when **input A/B are 8bits-typed float** for now."